Todo .go

```diff
@@ -64,16 +64,18 @@ func DeleteTodo(w http.ResponseWriter, r *http.Request) {
 	}
 	var indexToDelete int
 	for ind, todo := range todos {
-		if todo.Id == todoIdInt {
+		todoIdx, _ := strconv.Atoi(todo.Id)
+
+		if todoIdx == todoIdInt {
 			indexToDelete = ind // this is the index which has to be deleted from the list
 			break
 		}
 	}
 	var newTodoList []Todo
 	newTodoList = todos[:indexToDelete] //appending the list of todos to a new list excluding the one to be deleted
 	newTodoList = append(newTodoList, todos[(indexToDelete+1):]...)
-	_ = TodoCreateHelper(newTodoList[1:])      //inserting new list of todos into csv
-	json.NewEncoder(w).Encode(newTodoList[1:]) //returning response
+	_ = TodoCreateHelper(newTodoList)      //inserting new list of todos into csv
```

Comment:
see below, if you return the elements from position 1 you can skip it everywhere else